lib/ansible/modules/cloud/amazon/ec2_win_password.py

```diff
@@ -153,15 +161,12 @@ def main():
     else:
         try:
             with f:
-                key = RSA.importKey(f.read(), key_passphrase)
-        except (ValueError, IndexError, TypeError) as e:
+                key = load_pem_private_key(f.read(), key_passphrase, BACKEND)
```

Comment:
This will fail on Python 3 as f is opened as 'r' rather than 'rb' and load_pem_private_key requires bytes. key_passphrase must also be bytes -- if that path accepts utf8 `encode('utf8')` would be appropriate